### DISCUSSION TITLE
v1.1.4

### DISCUSSION THREAD

**Discussion by mxcl** [pkgxdev/pkgx#912] [Announcements]
**Full Changelog**: https://github.com/pkgxdev/pkgx/compare/v1.1.3...v1.1.4

<hr /><em>This discussion was created from the release <a href='https://github.com/pkgxdev/pkgx/releases/tag/v1.1.4'>v1.1.4</a>.</em>

**Comment by nhiphan2002** [comment]
tea That's so great

**Comment by adam50200** [comment]
It’s good

**Reply by ghost3015** [reply]
yep

**Comment by IpongJkBase** [comment]
Bullish af tea

**Comment by Claire3653** [comment]
I'm super bullish of tea! 
Cant wait for testnet!

**Comment by iman136938** [comment]
keep building . great job

**Comment by Ktkot1** [comment]
Bullish as here, great job

**Comment by Edotamara26** [comment]
That's great for projects 🔥

**Comment by minhdataaa** [comment]
I believe in a team with a lot of experience like the tea team
